pkg/requests/examples.go

```diff
@@ -140,6 +142,63 @@ func (ex *Examples) ChargeSucceeded() error {
 	return err
 }
 
+// SessionCreated creates a checkout session
+// https://stripe.com/docs/api/checkout/sessions/create?lang=curl
+func (ex *Examples) SessionCreated() error {
```

Comment:
Can you rename the method to `CheckoutSessionCompleted` and update the comment too?